tests/test_uci.py

```diff
@@ -0,0 +1,73 @@
+from chargemaster_parsers.parsers.uci import UCIChargeMasterParser
+from chargemaster_parsers.parsers import ChargeMasterEntry
+
+import pytest
+import json
+import io
+import pprint
+
+pp = pprint.PrettyPrinter()
+
+@pytest.fixture
+def parser():
+    yield UCIChargeMasterParser()
+
+def test_gross_charge_entry(parser):
+    row = {
+        "Gross Charges": [
+            {
+                "Itemcode": "00000001_7143",
+                "Description": "HB FINE NEEDLE ASPIRATION BX W/O IMG GDN 1ST LESION",
+                "CDM Revenue Code": "761",
```

Comment:
I was tempted to say these are NUBC revenue codes, but that doesn't seem to be the case. 0998 definitely is not for a skin test reading, nor is 761 for a fine needle aspiration :( Oh well. All's well